dotnet/hello_world/Program.cs

```diff
@@ -0,0 +1,210 @@
+                        _entry.RowKey = ByteString.CopyFromUtf8(rowKey);
+
+                        //adding entry to request
+                        request.Entries.Add(_entry);
+                    }
+
+                    BigtableClient.MutateRowsStream response = bigtableClient.MutateRows(request);
+
+                    Task write = CheckWrittenAsync();
+                    write.Wait();
+
+                    async Task CheckWrittenAsync()
+                    {
+                        IAsyncEnumerator<MutateRowsResponse> responseStream = response.ResponseStream;
+                        while (await responseStream.MoveNext())
+                        {
+                            MutateRowsResponse current = responseStream.Current;
+                        }
+                    }
+                    #endregion
+
+                    #region reading the first row
+                    Console.WriteLine("Reading the first row");
+                    int number = 1;
+                    for (int i = 0; i < number; i++)
+                    {
+                        BigtableByteString rowKey = String.Format("{0}{1}", _settings.RowKeyPrefix, i);
+                        Row row = bigtableClient.ReadRow(_tableClient, rowKey);
+
+                        Console.WriteLine($"\tRow key: {row.Key.ToStringUtf8()}, Value: {row.Families[0].Columns[0].Cells[0].Value.ToStringUtf8()}");
+                    }
+                    #endregion
+
+                    #region reading all rows
+
+                    BigtableClient.ReadRowsStream responseRead = bigtableClient.ReadRows(new ReadRowsRequest
+                    {
+                        TableNameAsTableName = new Google.Cloud.Bigtable.V2.TableName(ProjectId, InstanceId, _settings.TableName)
+                    });
+
+                    Task printRead = PrintReadRowsAsync();
+                    printRead.Wait();
+
+                    async Task PrintReadRowsAsync()
+                    {
+                        Console.WriteLine("Reading all rows using streaming");
+                        await responseRead.AsAsyncEnumerable().ForEachAsync(row =>
+                        {
+                            Console.WriteLine($"\tRow key: {row.Key.ToStringUtf8()}, Value: {row.Families[0].Columns[0].Cells[0].Value.ToStringUtf8()}");
+                        });
+                    }
+
+                    #endregion
+
+                    #region Deleting table
+                    Console.WriteLine($"Deleting table: {_settings.TableName}");
+
+                    bigtableTableAdminClient.DeleteTable(new Google.Cloud.Bigtable.Admin.V2.TableName(
+                        ProjectId,
+                        InstanceId,
+                        _settings.TableName), null);
+
+                    Console.WriteLine($"Table {_settings.TableName} deleted succsessfully");
```

Comment:
How do we know it's successful? Did we check the return code? Or did the previous step not throw an exception?

Since this is a "hello world" example, a new user of Bigtable (and the C# library) will want to know what they should be looking out for, what to check, how to do this, etc.

Reply:
I will use DoesTableExist method to verify that the table was indeed deleted successfully